linode/objects/support/ticket.py

```diff
@@ -57,25 +58,31 @@ def post_reply(self, description):
         return r
 
     def upload_attachment(self, attachment):
+        filename = None
         content = None
-        with open(attachment) as f:
-            content = f.read()
+        with open(attachment, 'rb') as f:
+            filename, content = os.path.basename(f.name), f.read()
 
         if not content:
             raise ValueError('Nothing to upload!')
 
         headers = {
-            "Authorization": "token {}".format(self._client.token),
-            "Content-type": "multipart/form-data",
+            "Authorization": "token {}".format(self._client.token)
         }
 
         result = requests.post('{}{}/attachments'.format(self._client.base_url,
                 SupportTicket.api_endpoint.format(id=self.id)),
-                headers=headers, files=content)
+                headers=headers, files={'file': (filename, content)})
 
         if not result.status_code == 200:
             errors = []
-            j = result.json()
+
+            # Try/catch here because we may not get back json if we get an error at the webserver level
```

Comment:
Yes you can. One very obvious example (and one I hit while testing) was 413 from nginx. While it looks like nginx is not configured correctly at this time (gives a 413 for what seems to be anything bigger than 1.5MB or something rather than 5MB)

This will be a problem even in the future if that limit is raised if you try to upload a file that is too large. (rare scenario, but still possible. instead of giving an api error, it gives a cannot parse json error. i figured i'd just catch all webserver level errors just in case there's something we missed)